pokete_data/attacks.py

```diff
@@ -485,21 +485,21 @@
         "ap": 30,
     },
     "eye_pick": {
-        "name": "Eye pick",
+        "name": "Eye Pick",
```

Comment:
For consistency, I would switch this to `Peck` as well:

```suggestion
        "name": "Eye Peck",
```